mcs/tools/mkbundle/mkbundle.cs

```diff
@@ -46,9 +47,12 @@ class MakeBundle {
 	static bool skip_scan;
 	static string ctor_func;
 	static bool quiet;
+	static string cross_target = null;
+	static string fetch_target = null;
 	static bool custom_mode = true;
 	static string embedded_options = null;
 	static string runtime = null;
+	static string target_server = "https://runtimes.go-mono.com/raw/";
```

Comment:
The SSL certificate for that domain is invalid.

Why aren't we using download.mono-project.com for this?